src/core/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,28 @@
+import { Checkbox } from '@material-ui/core';
+import React, { useState } from 'react';
+// import clsx from 'clsx';
+import CheckOutlinedIcon from '@material-ui/icons/CheckOutlined';
+import { useStyles } from './styles';
+import { BaseCheckboxProps } from './base';
+
+interface CheckBoxProps extends BaseCheckboxProps {
+  checked: boolean;
+}
```

Comment:
Checkbox API already provides `checked` thus there is no requirement of creating it again. [The API](https://material-ui.com/api/checkbox/#checkbox-api)